UM/ColorGenerator.py

```diff
@@ -1,6 +1,7 @@
 # Copyright (c) 2015 Ultimaker B.V.
 # Uranium is released under the terms of the AGPLv3 or higher.
 
+#pylint: disable=bad-whitespace
```

Comment:
I think this one isn't necessary any more, right, since https://github.com/Ultimaker/Uranium/pull/129/commits/bdc41ac5bb6fc2c105360c639341d27bd42eceb9?